### PR TITLE
Fixes Left Navigation Links

### PR DESCRIPTION
**Pull request by manishram**
Fixes: #564

Converted data inside `REPOSITORIES` `OPENING_CATEGORY_FILTERS` and `TEAM_NAME_FILTERS` into objects with `title` and `url` properties.

left

Account:
0f92e975101eb96cea80dce430d74efd3172fa229377818d467ded85dcf2ab6c

### CODE REVIEW

**Comment by lapstjup** [comment]
@manishram It would be good if @Hristijan95 reviews your changes since I guess these might be requested by him. Although instead of `any`, it would be better to create an interface for the title,url structure. Also I think - title,pathname sounds better. Confirm the same with him.

**Comment by Hristijan95** [comment]
Yep, I saw this yesterday and somehow forgot to comment. You need to create an interface instead of using any and bring back the generic <T> on the places where needed, I am not sure I understand why you changed all ComponentProps of FlatNavLinks to any. They need to be <T> & only create an new interface for the options type.
Also yeah, if it's no trouble, pathname sounds better.

File: src/utils/github.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ import {AMOUNT_COLOR, REPOSITORIES} from 'constants/github';
 import {BaseRelease, Issue, Release} from 'types/github';
 
 export const fetchGithubIssues = async (): Promise<Issue[]> => {
-  const promises = REPOSITORIES.map((repoName) =>
-    axios.get(`https://api.github.com/repos/thenewboston-developers/${repoName}/issues`),
+  const promises = REPOSITORIES.map((repoName: any) =>
```
Comment (jamessspanggg) [inline src/utils/github.ts:7]: opps, can remove the `any` here

Comment (manishram) [reply src/utils/github.ts]: done

**Comment by manishram** [comment]
This PR is in collaboration with @jamessspanggg . Thankyou James!!

**Comment by lapstjup** [comment]
@manishram and @jamessspanggg I have done some refactoring. The reason behind it - FlatNavLink doesn't make sense to be generic anymore since it's internal implementation depends on pathname and url fields which is obtained from NavOption.

File: src/containers/Teams/index.tsx [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+  {pathname: TeamName.frontEndDevelopers, title: 'Front-End Developers'},
+  {pathname: TeamName.kotlinSDK, title: 'Kotlin SDK'},
+  {pathname: TeamName.marketing, title: 'Marketing'},
+  {pathname: TeamName.payments, title: 'Payments'},
+  {pathname: TeamName.qa, title: 'QA'},
+  {pathname: TeamName.redditModerators, title: 'Reddit Moderators'},
+  {pathname: TeamName.research, title: 'Research'},
+  {pathname: TeamName.security, title: 'Security'},
+  {pathname: TeamName.slackManagers, title: 'Slack Manager'},
+  {pathname: TeamName.youtube, title: 'TouTube'},
```
Comment (jamessspanggg) [inline src/containers/Teams/index.tsx:29]: Should be `YouTube`

**Review by jamessspanggg — CHANGES_REQUESTED** [review]

**Comment by Hristijan95** [comment]
Looks really nice, I'm merging it

**Pull request merged**